platform/pensando/sonic-platform-modules-dpu/sonic_platform/interrupt.py

```diff
@@ -0,0 +1,184 @@
+# {C} Copyright 2023 AMD Systems Inc. All rights reserved
+#############################################################################
+# Pensando
+#
+# Module contains an implementation of APIs for fetch interrutps from docker-dpu
+# and listens to event generated by dpu. This get used by sonic-interruptd
```

Comment:
@shanshri i am not aware of sonic-interruptd, is this specific to AMD?

Reply:
@prgeor, it was outdated design for dpu-db-utild. Removed this code as no longer needed.